astropy/io/votable/tree.py

```diff
@@ -3087,6 +3094,66 @@ def _parse_fits(self, iterator, extnum, config):
 
         return array
 
+    def _parse_parquet(self, iterator, config):
+        """
+        Functionality to parse parquet files that are embedded
+        in VOTables.
+        """
+        # looks like votable already has a "Table" imported.
+        from astropy.table import Table as Table2
+
+        for start, tag, data, pos in iterator:
+            if tag == "STREAM":
+                if start:
+                    warn_unknown_attrs(
+                        "STREAM",
+                        data.keys(),
+                        config,
+                        pos,
+                        ["type", "href", "actuate", "encoding", "expires", "rights"],
+                    )
+                    href = data["href"]
+                    encoding = data.get("encoding", None)
+                else:
+                    break
+            else:  # in this case, there is no STREAM, hence no file linked.
+                href = ""
```

Comment:
This will cause an exception in the next line and therefore should be handled somehow.